src/main/java/io/pinecone/clients/Index.java

```diff
@@ -779,6 +780,207 @@ public DescribeIndexStatsResponse describeIndexStats(Struct filter) {
         return blockingStub.describeIndexStats(describeIndexStatsRequest);
     }
 
+    /**
+     * {@inheritDoc}
+     * <p> Example</p>
+     *  <pre>{@code
+     *      import io.pinecone.clients.Index;
+     *      import io.pinecone.clients.Pinecone;
+     *      import io.pinecone.proto.ListResponse;
+     *
+     *      Pinecone pc = new Pinecone.Builder(System.getenv("PINECONE_API_KEY")).build();
+     *      String indexName = "example-index";
+     *      Index index = pc.getIndexConnection(indexName);
+     *      ListResponse listResponse = index.list("example-namespace");
+     *      System.out.println(listResponse);
```

Comment:
I'd look at some of the other code snippets in this file and `AsyncIndex`. @ssmith-pc had suggested moving to a format where we drop some of the cruft around importing and instantiating the client to focus on the specifics of the operation. So for this one it would look something like:
```suggestion
     *      import io.pinecone.proto.ListResponse;
     *
     *     ...
     *
     *      ListResponse listResponse = index.list("example-namespace");
```

Reply:
Done!